### DISCUSSION TITLE
大家IOS下面用什么客户端？

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#1083] [General]
我用Vless协议xtls-rprx-direct 建了个服务，
Windows用 v2rayN 可以连很正常
IOS客户端是小火箭shadowroket版本 2.18.82（1318）连不上，找作者作者说他自己能连，一直在扯皮，我UUID没错的，因为v2rayn用二维码扫的，开了tls 和 xtls-direct

**Comment by Bosszhe666** [comment]
我用的quantumult这个显示
配置文件并未包含标识[SERVER]
小火箭也是节点更新不了，安卓可以用，iOS不能行啊

**Comment by 33lyl** [comment]
我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用

**Comment by ghost** [comment]
> 我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用

GRPC是什么？

**Comment by 33lyl** [comment]
> > 我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用
> 
> GRPC是什么？

恩？ 模式里面不是有h2 ，websocket，和grpc之类的

**Comment by ghost** [comment]
> > > 我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用
> > 
> > 
> > GRPC是什么？
> 
> 恩？ 模式里面不是有h2 ，websocket，和grpc之类的

我是普通的TCP都不行

**Comment by 33lyl** [comment]
> > > > 我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用
> > > 
> > > 
> > > GRPC是什么？
> > 
> > 
> > 恩？ 模式里面不是有h2 ，websocket，和grpc之类的
> 
> 我是普通的TCP都不行

你是用h2吗

**Comment by ghost** [comment]
> > > > > 我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用
> > > > 
> > > > 
> > > > GRPC是什么？
> > > 
> > > 
> > > 恩？ 模式里面不是有h2 ，websocket，和grpc之类的
> > 
> > 
> > 我是普通的TCP都不行
> 
> 你是用h2吗

不是，是TCP

**Comment by ghost** [comment]
> > > > > 我发现小火箭grpc好像不能用，扫描二维码的，电脑可以正常用
> > > > 
> > > > 
> > > > GRPC是什么？
> > > 
> > > 
> > > 恩？ 模式里面不是有h2 ，websocket，和grpc之类的
> > 
> > 
> > 我是普通的TCP都不行
> 
> 你是用h2吗

但我有HTTPS证书，不知道是不是这个原因，小火箭不行

**Reply by 33lyl** [reply]
我也有https证书，而且是用权威证书签发的
